src/Platforms/PostgreSQLPlatform.php

```diff
@@ -38,6 +40,10 @@
  */
 class PostgreSQLPlatform extends AbstractPlatform
 {
+    protected $types = [
```

Comment:
That variable is private in the parent class… is that intended?

Reply:
Ah should have been protected.